test/data/repos-repomd/repomd-modules-duplicit/repodata/repomd.xml

```diff
@@ -0,0 +1,18 @@
+<repomd xmlns="http://linux.duke.edu/metadata/repo">
+  <revision>1550000000</revision>
+  <data type="primary">
+    <checksum type="sha256">be3ca87f91b8c4bf54e5cb04bb36358744d8dbf9efd8910256bc27abb79a1cc6</checksum>
+    <open-checksum type="sha256">be3ca87f91b8c4bf54e5cb04bb36358744d8dbf9efd8910256bc27abb79a1cc6</open-checksum>
+    <location href="repodata/primary.xml" />
+    <timestamp>1597222003</timestamp>
+    <size>81503</size>
+    <open-size>864433</open-size>
+  </data>
+  <data type="modules">
+    <checksum type="sha256">18a868b0227d78b40ae22641390c496c943faafb3d67ea74603e2ab7819615da</checksum>
+    <open-checksum type="sha256">18a868b0227d78b40ae22641390c496c943faafb3d67ea74603e2ab7819615da</open-checksum>
+    <location href="repodata/modules.yaml" />
+    <timestamp>1641802880</timestamp>
+    <size>492</size>
+  </data>
+</repomd>
```

Comment:
There is missing new line at the end, but I am not sure whether it is fixable. @pkratoch What do you think?